artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/Message.java

```diff
@@ -726,20 +728,44 @@ default long getWholeMessageSize() {
 
    /**
     * @return Returns the message properties in Map form, useful when encoding to JSON
+    * @param valueSizeLimit
     */
-   default Map<String, Object> toPropertyMap() {
+   default Map<String, Object> toPropertyMap(int valueSizeLimit) {
```

Comment:
This is a breaking api change to users of core

Reply:
providing overrides with the limit leaves existing api as now. that is better. thanks!